pkg/cmd/server/origin/auth_config.go

```diff
@@ -83,11 +83,9 @@ func BuildAuthConfig(masterConfig *MasterConfig) (*AuthConfig, error) {
 	}
 
 	// Build the list of valid redirect_uri prefixes for a login using the openshift-web-console client to redirect to
-	// TODO: allow configuring this
-	// TODO: remove hard-coding of development UI server
 	assetPublicURLs := []string{}
 	if !options.DisabledFeatures.Has(configapi.FeatureWebConsole) {
-		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL, "http://localhost:9000", "https://localhost:9000"}
+		assetPublicURLs = []string{options.OAuthConfig.AssetPublicURL}
```

Comment:
> what are the steps that would need to be run?

after the master starts, add `https://localhost:9000` to the `redirectURIs` list
```
oc patch oauthclient/openshift-web-console -p '{"redirectURIs":["https://localhost:9000"]}'
```